docs/io.cozy.contacts.md

```diff
@@ -56,6 +57,7 @@ The `io.cozy.contacts` doctype is loosely based on the [vCard RFC](https://tools
   - `type?`: {string} A user-provided localized type (example: `"Personnal"`, `"Profressional"`, `"Other"`)
   - `label?`: {string} A keyword to identify the type, must be `work|home`
   - `primary?`: {boolean} Indicates a preferred-use instance
+- `url?`: {string} Deprecated. Cozy instance url. Use `cozy?` array of cozy instances above.
```

Comment:
small nit: s/above/instead to avoid order requirement

Reply:
Done.